remove_mobile_app_links/views/hr_expense.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record
+        id="hr_expense.hr_expense_actions_my_unsubmitted"
```

Comment:
Here you fixed the view `hr_expense_actions_my_unsubmitted` but the same fix should be done also for `hr_expense_actions_my_all`